analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -797,6 +796,79 @@ def add_arguments_to_parser(parser):
         func=main, func_process_config_file=cmd_config.process_config_file)
 
 
+def is_analyzer_config_valid(analyzer_conf: List[AnalyzerConfig]) -> bool:
```

Comment:
You exit this function upon finding the very first invalid config. You could instead emit an error for all invalid configs, and exit in the end (this could save precious failed reruns of an ill-configured CI job).